### PR TITLE
chore: release

### PR DESCRIPTION
**Pull request by aymeric-giraudet**
**Summary**

**Result**

### CODE REVIEW

File: packages/create-instantsearch-app/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-instantsearch-app",
-  "version": "6.4.2",
+  "version": "6.5.0",
```
Comment (aymeric-giraudet) [inline packages/create-instantsearch-app/package.json:3]:
```suggestion
  "version": "7.0.0",
```

File: packages/create-instantsearch-app/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [6.5.0](https://github.com/algolia/instantsearch/compare/create-instantsearch-app@6.4.2...create-instantsearch-app@6.5.0) (2023-08-08)
```
Comment (aymeric-giraudet) [inline packages/create-instantsearch-app/CHANGELOG.md:6]:
```suggestion
# [7.0.0](https://github.com/algolia/instantsearch/compare/create-instantsearch-app@6.4.2...create-instantsearch-app@7.0.0) (2023-08-08)
```

File: packages/instantsearch.js/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,22 @@
...
+
+
+### Features
+
+* rename packages ([#5787](https://github.com/algolia/instantsearch.js/issues/5787)) ([c133170](https://github.com/algolia/instantsearch.js/commit/c133170e563592dfc15a95daced1f8447327a09a))
+
+
+
+
+
```
Comment (aymeric-giraudet) [inline packages/instantsearch.js/CHANGELOG.md:21]:
```suggestion
```

File: examples/js/calendar-widget/package.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "algoliasearch": "4.14.3",
-    "instantsearch.js": "4.56.8"
+    "instantsearch.js": "4.57.0"
```
Comment (aymeric-giraudet) [inline examples/js/calendar-widget/package.json:12]:
```suggestion
    "instantsearch.js": "4.56.9"
```

File: examples/js/e-commerce-umd/package.json [hunk 5]
```diff
@@ -10,7 +10,7 @@
   "browserslist": "firefox 68, chrome 78, IE 11",
   "dependencies": {
     "algoliasearch": "4.14.3",
-    "instantsearch.js": "4.56.8"
+    "instantsearch.js": "4.57.0"
```
Comment (aymeric-giraudet) [inline examples/js/e-commerce-umd/package.json:13]:
```suggestion
    "instantsearch.js": "4.56.9"
```

File: examples/js/e-commerce/package.json [hunk 6]
```diff
@@ -10,7 +10,7 @@
   "browserslist": "firefox 68, chrome 78, IE 11",
   "dependencies": {
     "algoliasearch": "4.14.3",
-    "instantsearch.js": "4.56.8"
+    "instantsearch.js": "4.57.0"
```
Comment (aymeric-giraudet) [inline examples/js/e-commerce/package.json:13]:
```suggestion
    "instantsearch.js": "4.56.9"
```

File: packages/react-instantsearch-router-nextjs/package.json [hunk 7]
```diff
@@ -49,8 +49,8 @@
     "test:e2e:saucelabs": "start-server-and-test test:start-server 3000 'wdio run ./wdio.saucelabs.conf.cjs'"
   },
   "dependencies": {
-    "instantsearch.js": "4.56.8",
-    "react-instantsearch-core": "7.0.0"
+    "instantsearch.js": "4.57.0",
```
Comment (aymeric-giraudet) [inline packages/react-instantsearch-router-nextjs/package.json:52]:
```suggestion
    "instantsearch.js": "4.56.9",
```

File: packages/react-instantsearch/package.json [hunk 8]
```diff
@@ -48,8 +48,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.1.2",
-    "instantsearch.js": "4.56.8",
-    "react-instantsearch-core": "7.0.0"
+    "instantsearch.js": "4.57.0",
```
Comment (aymeric-giraudet) [inline packages/react-instantsearch/package.json:51]:
```suggestion
    "instantsearch.js": "4.56.9",
```

File: tests/common/package.json [hunk 9]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "@instantsearch/tests",
   "private": true,
-  "version": "1.22.0",
+  "version": "1.23.0",
   "description": "Common tests for all InstantSearch flavors.",
   "dependencies": {
-    "instantsearch.js": "4.56.8"
+    "instantsearch.js": "4.57.0"
```
Comment (aymeric-giraudet) [inline tests/common/package.json:7]:
```suggestion
    "instantsearch.js": "4.56.9"
```

File: tests/mocks/package.json [hunk 10]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "@instantsearch/mocks",
-  "version": "1.22.0",
+  "version": "1.23.0",
   "private": true,
   "dependencies": {
     "algoliasearch-helper": "3.14.0",
-    "instantsearch.js": "4.56.8"
+    "instantsearch.js": "4.57.0"
```
Comment (aymeric-giraudet) [inline tests/mocks/package.json:7]:
```suggestion
    "instantsearch.js": "4.56.9"
```

File: tests/utils/package.json [hunk 11]
```diff
@@ -1,13 +1,13 @@
...
-  "version": "1.11.0",
+  "version": "1.12.0",
   "private": true,
   "dependencies": {
     "@types/enzyme": "3.1.15",
     "enzyme": "3.11.0",
     "enzyme-adapter-preact-pure": "2.2.0",
     "enzyme-to-json": "3.6.2",
-    "instantsearch.js": "4.56.8",
+    "instantsearch.js": "4.57.0",
```
Comment (aymeric-giraudet) [inline tests/utils/package.json:10]:
```suggestion
    "instantsearch.js": "4.56.9",
```

**Review by Haroenv — CHANGES_REQUESTED** [review]
let's wait until FX-2554 is done

**Review by Haroenv — APPROVED** [review]

**Pull request merged**